nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
+     license agreements. See the NOTICE file distributed with this work for additional
+     information regarding copyright ownership. The ASF licenses this file to
+     You under the Apache License, Version 2.0 (the "License"); you may not use
+     this file except in compliance with the License. You may obtain a copy of
+     the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+     by applicable law or agreed to in writing, software distributed under the
+     License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+     OF ANY KIND, either express or implied. See the License for the specific
+     language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+        <modelVersion>4.0.0</modelVersion>
+
+        <parent>
+                <groupId>org.apache.nifi</groupId>
+                <artifactId>nifi-email-bundle</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+        </parent>
+
+        <artifactId>nifi-email-processors</artifactId>
+        <packaging>jar</packaging>
+
+        <dependencies>
+                <dependency>
+                        <groupId>org.apache.nifi</groupId>
+                        <artifactId>nifi-api</artifactId>
+                </dependency>
+                <dependency>
+                        <groupId>org.apache.nifi</groupId>
+                        <artifactId>nifi-processor-utils</artifactId>
+                </dependency>
+                <dependency>
+                        <groupId>javax.mail</groupId>
+                        <artifactId>mail</artifactId>
+                        <version>1.4.7</version>
```

Comment:
javax.mail mail is already defined in the dependency management section of the root pom[1] so the "version" here can be removed.

[1] https://github.com/apache/nifi/blob/2c95abbe20876d30ccd14fe80e766fd509a67d1a/pom.xml#L270-L270